lib/semantics/resolve-names.cc

```diff
@@ -1116,20 +1116,18 @@ bool OmpVisitor::NeedsScope(const parser::OpenMPBlockConstruct &x) {
   const auto &beginBlockDir{std::get<parser::OmpBeginBlockDirective>(x.t)};
   const auto &beginDir{std::get<parser::OmpBlockDirective>(beginBlockDir.t)};
   switch (beginDir.v) {
-  case parser::OmpBlockDirective::Directive::TargetData:
-  case parser::OmpBlockDirective::Directive::Master:
+  case parser::OmpBlockDirective::Directive::TargetData: [[fallthrough]];
+  case parser::OmpBlockDirective::Directive::Master: [[fallthrough]];
```

Comment:
There are probably hundreds of other case labels like this where this annotation would apply. What makes these two special?